JSDemos/Demos/Tooltip/Overview/jQuery/description.md

```diff
@@ -0,0 +1,15 @@
+The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to the element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
+
+## Show and Hide the Tooltip
+
+To show and hide the Tooltip when certain events are triggered, set the [showEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/showEvent/) and [hideEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/hideEvent/) properties. These properties can accept several events at once as well as an object.
```

Comment:
"Several" is a colloquialism. Use "multiple" instead. These words have identical meaning, however, "multiple" is more suitable to documentation.

```suggestion
To show and hide the Tooltip in response to certain events, specify the [showEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/showEvent/) and [hideEvent](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/hideEvent/) properties. These properties can accept multiple events at once as well as an object.
```